src/components/List/Body/Body.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types'; // ES6
+import classes from './Body.module.css';
+
+const Body = (props) => {
+
+    var PropTypes = require('prop-types'); // ES5 with npm
+
+    return (
+        <div className={classes.resultsSection}>
+
+            <div className={classes.header}>
+                <h2 className={classes.text}>Results</h2>
+            </div>
+
+            <div className={classes.content}>
+                {Array(props.totalCards).fill().map((row, i) =>
```

Comment:
I think it's better to loop on the props.filteredDataArray directly

Reply:
done